src/main/java/seedu/address/model/property/Properties.java

```diff
@@ -1,4 +1,4 @@
-package seedu.address.model.role;
+package seedu.address.model.property;
```

Comment:
We may not need this class since it was originally used to represent a list of `Properties` that a seller is in charge of - but we no longer make use of a `Seller` class anymore.